### ISSUE TITLE
handler.c:142:14: error: implicit declaration of function 'nabi_ic_lookup_keysym' is invalid in C99

### ISSUE THREAD

**Issue by barracuda156** [libhangul/nabi#4]
So I am updating `nabi` in Macports too, and Intel build fails on MacOS 12:
```
handler.c:142:14: error: implicit declaration of function 'nabi_ic_lookup_keysym' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
keysym = nabi_ic_lookup_keysym(ic, kevent);
              ^
```
Log here: https://github.com/macports/macports-ports/runs/7888565908
(On 10.6.8 Rosetta I did not get this error.)

Perhaps I could just cut out `-Werror` flag, but maybe it is better to fix it some other way.

**Comment by choehwanjin** [comment]
This issue has already been fixed in git version(11f55b4de0db41faff21f35f22b4ff17a52ef741).
I will release new version soon.

**Comment by barracuda156** [comment]
> This issue has already been fixed in git version(11f55b4de0db41faff21f35f22b4ff17a52ef741).
> 
> I will release new version soon.

@choehwanjin 
UPD: I decided to just use a patch from your commit to existing release version, and it solved the problem. (Will be easier to update to the next release, just dropping a patch.)

Thank you!
